### PR TITLE
Use HIPCMatrix R package

### PR DESCRIPTION
**Pull request by helenmiller16**
## Description

Update HIPCMatrix pipeline to use HIPCMatrix R package: https://github.com/RGLab/HIPCMatrix 
Installation instructions and Notion ticket: https://www.notion.so/rglab/HIPMatrix-f157b6c1db9e4c8e85ac6b4d0daaccc7#adb1ce2d90854e8cb35e02c33b99ef44

## PR Checklist for reviewer

- [ ] Build it on your own local instance and test new features or changes.  
- [ ] Pass UITesting on your local instance on the matching UITesting branch.  
- [ ] It is functional and meets the requirements?   
- [ ] Are any shorthand variable names are included in the [Glossary](https://www.notion.so/rglab/Glossary-of-variable-abbreviations-e205838b1f534abc903fa8c2228a6d7f)?  
- [ ] Does it include descriptive code comments?   
- [ ] Does it follow the [Style Guide](https://github.com/RGLab/LabKeyModules/#style-guide) for any languages included?  
- [ ] Is it well-documented?
    * Module-specific README lists out any dependencies  
    * Sufficient instructions for setup  
    * Links to any relevant LabKey documentation   
- [ ] Are any new features tested via UITesting or jest?  
- [ ] Does it have any new dependencies? Is it justifiable?  
- [ ] Is there code duplication that could be reduced?  
- [ ] Are there performance improvements that could be made?

### CODE REVIEW

**Comment by helenmiller16** [comment]
@juyeongkim PR ready for review. I updated HIPCMatrix package with more documentation around meta data and how that works, added mean vs max option for summarization, github actions run to test on windows and linux, and tests for duplicated biosample ids.

File: HIPCMatrix/pipeline/tasks/create-matrix.R [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
 # DEPENDENCIES
 #--------------------------------
-
-# NOTES:
-# 1. Wrapper function, separated for sourcing when running on CL
-# 2. other libraries loaded in createMatrixWrapper.R
-# 3. Using full path because script is copied to study-specific directory for
-# running from UI.
-LKModules <- "/labkey/git/LabKeyModules"
-source(file.path(LKModules, "HIPCMatrix/pipeline/tasks/runCreateMx.R"))
```
Comment (juyeongkim) [inline HIPCMatrix/pipeline/tasks/create-matrix.R:11]: This is my favorite deletion.

File: HIPCMatrix/assay/ExpressionMatrix/queries/SelectedRuns.query.xml [hunk 2]
```diff
@@ -34,6 +34,9 @@
                   <column columnName='download_link'>
                     <url>/_webdav/Studies/${folder/name}/%40files/analysis/exprs_matrices/${name}.tsv</url>
                   </column>
+                  <column columnName="Version">
+                    <url>https://github.com/RGLab/HIPCMatrix/tree/${hash}</url>
+                  </column>
```
Comment (juyeongkim) [inline HIPCMatrix/assay/ExpressionMatrix/queries/SelectedRuns.query.xml:39]: This is my favorite addition.

**Review by juyeongkim — APPROVED** [review]
💯 Great work!

**Pull request merged**